x-pack/plugins/security_solution/public/common/components/exceptions/types.ts

```diff
@@ -56,6 +58,27 @@ export interface Comment {
   comment: string;
 }
 
+export enum ToggleId {
+  DETECTION_ENGINE = 'detection',
+  ENDPOINT = 'endpoint',
+}
+
+export interface RuleExceptionList {
+  id: string | null;
+  type: string | null;
+  namespaceType: NamespaceType | null;
+}
```

Comment:
Is it necessary for these to be `null` as well? Can the default values in the reducer just be `null` instead since they're actually the un-set values?

https://github.com/elastic/kibana/blob/8de74693c27ccde058d0df0151e5e00615b0702b/x-pack/plugins/security_solution/public/common/components/exceptions/viewer/reducer.ts#L15-L16

Reply:
This was removed in the refactor.